docs/sources/tempo/configuration/_index.md

```diff
@@ -376,6 +376,9 @@ metrics_generator:
         # How long to wait when flushing samples on shutdown
         [remote_write_flush_deadline: <duration> | default = 1m]
 
+        # Whether to remove X-Scope-OrgID header in remote write requests
+        [remote_write_remove_org_id_header: <bool> | default = false]
```

Comment:
would prefer an "add org id" setting that defaults to true

Reply:
@joe-elliott I have changed the config name to remote_write_add_org_id_header. https://github.com/grafana/tempo/pull/2974/commits/2ac2622c01979c901674f3e7610e3ed5af5819db